### PR TITLE
Desktop features 

### PR DESCRIPTION
**Pull request by GloireIrenge**
In this pull request I:
- Edit the portfolio mobile version and make it responsive desktop version
- Add some animations
- Make the Dekstop version looks like the Figma template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,10 +9,29 @@
...
+                    
+                    <div class="nav-links">
+                      <a href="#">Portfolio</a>
+                      <a href="#">About</a>
+                      <a href="#">Contact</a>
+                      <a href="#"><img src="imgs/mail.svg" alt="Mail"></a>
+                      
+                    </div>
+                  </div>
         <div class="body-class">
```
Comment (GoddessNgozi) [inline index.html:35]: Header should contain logo and top navigational links. Please remove the h1, span, p and social icons.

Comment (GoddessNgozi) [inline index.html:35]: Screenshot 2022-07-08 at 11 50 09

Works section should be placed inside of the main tag. Important sections in a website are placed inside a main tag.

Comment (GoddessNgozi) [inline index.html:35]: Screenshot 2022-07-08 at 11 58 13

Kindly place the My_footer div inside of a footer tag. It is important to use semantic tags to section your work for good/readability, for screen readers and SEO.

**Review by GoddessNgozi — CHANGES_REQUESTED** [review]

File: main.css [hunk 2]
```diff
@@ -407,11 +609,195 @@ input[type=submit] {
...
+  grid-template-columns: 1fr 1fr;
+  gap: 60px;
+  align-items: center;
+  padding-bottom: 40px;
+  padding-right: 10%;
+  padding-left: 10%;
+}
+
+/* Responsive */
+@media screen and (max-width: 980px) {
```
Comment (topeogunleye) [inline main.css:657]: - [ ] I kindly suggest that you make your project styling mobile-first. That would mean that you need to use `min-width` to style for desktops and not `max-width` to style for mobile.

File: index.html [hunk 3]
```diff
@@ -9,53 +9,100 @@
...
+                    <div class="works-title">
+                       <h1>My Recent Works</h1>
+                    </div>
+                      <div class="multi-post">
+                        <div class="line"></div>
+                      </div></article>
+          </div>
+        
+
+            <article class="featured">
```
Comment (topeogunleye) [inline index.html:78]: - [ ] I kindly suggest that you add your `see project` button that takes you to the Live Demo of the featured project or to the GitHub repository.

| Your Design  | Template Design |
| ------------- | ------------- |
| image |  image|

File: index.html [hunk 4]
```diff
@@ -147,36 +194,25 @@ <h2>Profesional Art Printing Data</h2>
                     <button class="big-button" type="button">See Project</button>
                 </div>  
 
-                <div class="grid-item">
-                    <div class="multi-post2">
```
Comment (topeogunleye) [inline index.html:151]: - [ ] As written in the project requirements. I kindly suggest that you ensure that when a project is hovered on with the mouse:
-  the description and dark opacity disappear
-  and a See Project button appears to the bottom of the image and takes the whole image width.
-   Your `See Project` button should link to the Live Demo of the project or to the GitHub repository.

- [ ] [OPTIONAL] Please ensure that your padding for your project is similar to the template design. Kindly see the images below

| Your Design  |Template Design |
| ------------- | ------------- |
| image | image |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @GloireIrenge,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**